src/Kovan-DssSpell.t.sol

```diff
@@ -1179,6 +1179,35 @@ contract DssSpellTest is DSTest, DSMath {
             calc_step:    0,
             calc_cut:     0
         });
+        afterSpell.collaterals["MATIC-A"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      10 * MILLION,
+            aL_gap:       3 * MILLION,
+            aL_ttl:       8 hours,
+            line:         3 * MILLION, // TODO: is this correct? currently setting this according to dc-iam gap
+            dust:         10000, // TODO: seeing other tests set this to 100, maybe for supporitng tests
+            pct:          300,
+            mat:          17500,
+            liqType:      "clip",
+            liqOn:        true,
+            chop:         1300,
+            cat_dunk:     0,
+            flip_beg:     0,
+            flip_ttl:     0,
+            flip_tau:     0,
+            flipper_mom:  0,
+            dog_hole:     3 * MILLION, // TODO: seeing other tests set this to 5 * THOUSAND, maybe for supporitng tests
```

Comment:
yeah, let's change this to 5000 for consistency. You'll need to update the spell for this one.

Reply:
done